db/migrations/0009_update_retrieval_logs_null.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE retrieval_logs RENAME TO retrieval_logs_old;
+CREATE TABLE retrieval_logs (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  timestamp DATETIME NOT NULL,
+  owner_address TEXT, -- DROP NOT NULL constraint
+  client_address TEXT NOT NULL,
+  response_status INTEGER, -- DROP NOT NULL constraint
+  egress_bytes INTEGER,
+  cache_miss BOOLEAN, -- DROP NOT NULL constraint
+  fetch_ttfb INTEGER,
+  worker_ttfb INTEGER,
+  request_country_code TEXT,
+  fetch_ttlb INTEGER
```

Comment:
```suggestion
  fetch_ttlb INTEGER,
  proof_set_id TEXT
```

We'd also need to rename the file to `0010_update_retrieval_logs_null.sql`